rpcs3/Emu/RSX/RSXThread.h

```diff
@@ -59,56 +55,54 @@ namespace rsx
 			return this->ea[offs >> 20] | (offs & 0xFFFFF);
 		}
 
-		template<bool IsFullLock>
+		template <bool IsFullLock, uint Stride>
```

Comment:
This is worrying.
When would we ever want Stride to not be 1 and it still be a safe operation?

Reply:
This change was done to make accurate RSX FIFO and accurate RSX reservations as much as uncorrelated as possible with reservations keeping one lock every 1MB and FIFO keeping locks every 8k. This is a performance adjustment because access RSX FIFO access may false lock if the SPU writes anywhere mildly near the FIFO GET pointer which is undesirable.

Reply:
For the record, I'm ok with merging the two options into one.